client/src/views/Authentication/AuthForgot.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="q-pa-md">
+    <div class="q-gutter-y-sm column fixed register" style="width: 300px">
+
+      <p style="text-align: center; margin-top:20px; margin-bottom: 0"><strong>Forgot Password?</strong></p>
+
+      <p style="text-align: center; margin-top:20px; margin-bottom: 0">Enter the email address associated with your account</p>
+
+      <p style="text-align: center; margin-top:20px">We will email you a link to reset your password</p>
```

Comment:
Inline styling, DRY(Don't repeat yourself) - "text-align: center; margin-top:20px" you used 3x times.